Formula/makeself.rb

```diff
@@ -1,5 +1,5 @@
 class Makeself < Formula
-  desc "Make self-extractable archives on UNIX"
+  desc "Script that generates a self-extractable compressed tar archive"
```

Comment:
```suggestion
  desc "Generates a self-extractable compressed tar archive"
```

Reply:
I think "self-extracting" is the commonly-used adjective for such archives.

Reply:
@gromgit Good catch!